src/node.cc

```diff
@@ -3059,7 +3059,7 @@ static void DebugPortGetter(Local<Name> property,
 #if HAVE_INSPECTOR
   if (port == 0) {
     Environment* env = Environment::GetCurrent(info);
-    if (env->inspector_agent()->IsStarted())
+    if (env->inspector_agent()->io())
```

Comment:
DRY: `if (auto io = env->inspector_agent()->io()) port = io->port();`?

Should it check `io->IsStarted()`?  The inspector has a venerable multitude of IsStarted methods...